enterprise/server/workflow/config/config.go

```diff
@@ -45,3 +53,53 @@ func NewConfig(r io.Reader) (*BuildBuddyConfig, error) {
 	}
 	return cfg, nil
 }
+
+// GetDefault returns the default workflow config, which tests all targets
+// when pushing to the given target branch, sending build events to BuildBuddy.
+func GetDefault(targetBranch, besBackend, besResultsURL, apiKey string) *BuildBuddyConfig {
+	return &BuildBuddyConfig{
```

Comment:
why do we call this BuildBuddyConfig instead of WorkflowConfig?

Reply:
No strong reason, other than that the file name is `buildbuddy.yaml` and presumably this is where all buildbuddy-related configuration would go. I can send a follow-up PR to rename it, if you think it's less confusing that way